src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ConstantPool.java

```diff
@@ -170,6 +170,17 @@ interface BootstrapMethodInvocation {
         List<JavaConstant> getStaticArguments();
     }
 
+    /**
```

Comment:
This is very confusing documentation. It's not clear what index or entry you are referring to each time you say "index". Please provide a concrete example.

Reply:
I have rewritten the documentation, I hope it is clearer. I also added a second argument to be able to access different different static arguments on the same bootstrap specifier.

Reply:
It still needs an example showing how this API should be used. Please add a complete (pseudo-code) example to the javadoc of `BootstrapMethodInvocation`.